### DISCUSSION TITLE
.deb installation package

### DISCUSSION THREAD

**Discussion by norbertcserpnyak16** [ppy/osu#14620] [Ideas] [answered]
AppImage has one problem, it doesn't save the music you put in and you have to put it in again and again and again when you open it again.

So a .deb package would be useful.

**Comment by gagahpangeran** [comment, marked as answer]
> AppImage has one problem, it doesn't save the music you put in and you have to put it in again and again and again when you open it again.

What do you mean by this?

All of the beatmaps and game data is saved in `~/.local/share/osu` by default. It's not removed when the appimage is removed or replaced (unless you remove it manually).

> So a .deb package would be useful.

We only maintain and release appimage file because it's distro agnostic. We don't have resource to maintain any distro spesific package file.
